### PR TITLE
feat(docs): add CI Spell checks

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
## Proposed Changes

_Adds CI Spell Checks._

## Linked Issues

_[939](https://github.com/fetchai/docs/issues/939)_

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Content update.
- [ ] Bug fix (non-breaking change that fixes an issue).
- [x] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/docs/blob/main/CONTRIBUTING.md) guide
- [x] Checks and tests pass locally

## Further comments

_[if this is a relatively large or complex change, kick off a discussion by explaining why you chose the solution you did, what alternatives you considered, etc...]_

### CODE REVIEW

File: .github/workflows/.wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,923 @@
...
+NetworkConfigError
+TotalSupply
+QueryTotalSupplyRequest
+NetworkConfig
+InflationRate
+dorado
+fetchai
+testnet
+Fetch.ai
+fetch.ai
```
Comment (devjsc) [inline .github/workflows/.wordlist.txt]: Maybe Fetch.ai should always be capitalised?

File: .github/workflows/.wordlist.txt [hunk 2]
```diff
@@ -0,0 +1,923 @@
...
+CommandLine
+InsufficientFeesError
+ParsedUrl
+CosmPy
+Cosmpy
+compounder
+blockchain
+Blockchain
+Compounder
+uAgents
```
Comment (devjsc) [inline .github/workflows/.wordlist.txt]: on uAgent/uAgents should be allowed?

File: .github/workflows/.wordlist.txt [hunk 3]
```diff
@@ -0,0 +1,923 @@
...
+LegacyBridgeSwaps
+postgraphile
+Postgraphile
+enum
+UIs
+UI
+uis
+Dorado
+Fetchhub
+fetchhub
```
Comment (devjsc) [inline .github/workflows/.wordlist.txt]: Fetch should always be capitalised?

File: .github/workflows/.wordlist.txt [hunk 4]
```diff
@@ -0,0 +1,923 @@
...
+HSM
+DDOS
+DDoS
+fetchai
+ASI
+asi
+Asi
+FET
+Fet
+fet
```
Comment (devjsc) [inline .github/workflows/.wordlist.txt]:
```suggestion

```

File: .github/workflows/.wordlist.txt [hunk 5]
```diff
@@ -0,0 +1,923 @@
...
+azzhthsz
+fajzy
+gge
+hh
+jhwtat
+klzf
+lk
+njmfn
+qgp
+qvhlqy
```
Comment (devjsc) [inline .github/workflows/.wordlist.txt]: this is not a word 😅

**Review by devjsc — CHANGES_REQUESTED** [review]

**Comment by devjsc** [comment]
```
Misspelled words:
<htmlcontent> ./pages/apis/agentverse/hosting.mdx: html>body>p
--------------------------------------------------------------------------------
cuvclpj
dfhywtt
efedf
esauaegcm
gjqt
gzdd
uhk
wmc
xazrdyzgap
--------------------------------------------------------------------------------
```


This section is failing in pages/apis/agentverse/hosting.mdx

```
json filename="Sample Agent Object"
        {
            "name": "My first agent",
            "address": "agent1q2dfhywtt8xazrdyzgap6gzdd7uhk4e0wmc3gjqt42esauaegcm8cuvclpj",
            "running": false,
            "compiled": true,
            "revision": 7,
            "code_digest": "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef",
            "wallet_address": "fetch1dtwgzm6km4394erexa8ka05wva306wt9cc3mwk"
        }
```
        
Which tells us that spellcheck.yml is not actually ignoring code 😅

File: dictionaries/code_warnings_known.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+10px
```
Comment (devjsc) [inline dictionaries/code_warnings_known.txt:1]: We don't use this, do we?

File: dictionaries/custom_dict.txt [hunk 7]
```diff
@@ -0,0 +1,634 @@
...
+DDOS
+DDoS
+DENOM
+DNS
+DataBuyingAgent
+Databricks
+Datetime
+DeFi
+Debian
+Decentr
```
Comment (devjsc) [inline dictionaries/custom_dict.txt:65]: what is this word? 😅

File: dictionaries/custom_dict.txt [hunk 8]
```diff
@@ -0,0 +1,634 @@
...
+NativeTransfers
+NetworkConfig
+NetworkConfigError
+NextGeneration
+Nominatim
+NotFoundError
+November
+OAuth
+OfflineMessageTableStrategy
+Ohh
```
Comment (devjsc) [inline dictionaries/custom_dict.txt:164]: Oh?

File: dictionaries/code_warnings_known.txt (hunk 6 above)
Comment (FelixNicolaeBucsa) [reply dictionaries/code_warnings_known.txt:1]: No, we are not.

File: dictionaries/custom_dict.txt (hunk 7 above)
Comment (FelixNicolaeBucsa) [reply dictionaries/custom_dict.txt:65]: Decentr?

It is in wallet docs

File: dictionaries/custom_dict.txt (hunk 8 above)
Comment (FelixNicolaeBucsa) [reply dictionaries/custom_dict.txt:164]: Yup. It is in a code output

File: dictionaries/custom_dict.txt (hunk 7 above)
Comment (FelixNicolaeBucsa) [reply dictionaries/custom_dict.txt:65]: Double checking this one

Comment (devjsc) [reply dictionaries/custom_dict.txt:65]: It's a browser I have never heard of

File: dictionaries/custom_dict.txt (hunk 8 above)
Comment (devjsc) [reply dictionaries/custom_dict.txt:164]: Ohh, I have found this.

File: dictionaries/custom_dict.txt (hunk 7 above)
Comment (FelixNicolaeBucsa) [reply dictionaries/custom_dict.txt:65]: It is in pages/guides/fetch-network/fetch-wallet/web-wallet/get-started.mdx

**Pull request merged**